### PR TITLE
Make property names React compatible(?)

### PR DESCRIPTION
**Pull request by jwalter**
Following the patter, hoping it does not break anything... These changes were necessary for me to build successfully.

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
#### 📝Changed routes:


| [**/**](https://randomcodegenerator-lol-lvyldhffw-whitep4nth3r.vercel.app/) |
|:-:|
|<a href="https://randomcodegenerator-lol-lvyldhffw-whitep4nth3r.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/OLNWRj5deK5ulnLDQrCoHNFiJ8ZkOCYjRTKxLIWKCA6zq2BXssFJiX_IlZCmIySI4zcZ0cxVttkiMwIPwjScAQ.png">(view full size)</a> |







Commit 26e616766627c16815ec57769dbadb2e581ccfe8 (https://randomcodegenerator-lol-lvyldhffw-whitep4nth3r.vercel.app).

File: components/CopyIcon/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function CopyIcon() {
       <path
         fill="currentColor"
         d="M433.941 65.941l-51.882-51.882A48 48 0 0 0 348.118 0H176c-26.51 0-48 21.49-48 48v48H48c-26.51 0-48 21.49-48 48v320c0 26.51 21.49 48 48 48h224c26.51 0 48-21.49 48-48v-48h80c26.51 0 48-21.49 48-48V99.882a48 48 0 0 0-14.059-33.941zM266 464H54a6 6 0 0 1-6-6V150a6 6 0 0 1 6-6h74v224c0 26.51 21.49 48 48 48h96v42a6 6 0 0 1-6 6zm128-96H182a6 6 0 0 1-6-6V54a6 6 0 0 1 6-6h106v88c0 13.255 10.745 24 24 24h88v202a6 6 0 0 1-6 6zm6-256h-64V48h9.632c1.591 0 3.117.632 4.243 1.757l48.368 48.368a6 6 0 0 1 1.757 4.243V112z"
-        class=""
+        className=""
```
Comment (whitep4nth3r) [inline components/CopyIcon/index.js:13]: You can actually just remove this completely.

File: pages/index.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ export default function Home({ languages, languageKey, code, lines, contributors
                 <div className="result__topBar">
                   <WindowIcons />
                   <button type="button" className="copyButton" onClick={copyCode}>
-                    <span class="copyButton__iconContainer">
+                    <span className="copyButton__iconContainer">
```
Comment (whitep4nth3r) [inline pages/index.js:123]: Thank you for cleaning up my mess!

File: components/CopyIcon/index.js (hunk 1 above)
Comment (jwalter) [reply components/CopyIcon/index.js]: Done

**Pull request merged**